integrations/src/test/resources/io/jenkins/plugins/casc/MailerTest.yml

```diff
@@ -1,5 +1,5 @@
 unclassified:
   mailer:
     replyToAddress: do-not-reply@acme.org
-    smtpHost: smtp.acme.org
-    smtpPort: 4441
+    smtpHost: "smtp.acme.org"
+    smtpPort: "4441"
```

Comment:
this is not ideal, I think I looked at it earlier and was going to be difficult to fix though, worried about breaking PCT again with this 😢 

Thoughts @oleg-nenashev / @casz ?

Reply:
Quoting should not be needed indeed, best figure out why it fails without quoting

Reply:
I guess the reason I quoted was because the schema validation failed, it's time we developed our own validation engine using hamcrest validation